src/main/java/edu/hawaii/its/groupings/service/UhUuidCheckerService.java

```diff
@@ -23,4 +24,9 @@ public boolean isValidUhUuid(String uhUuid, String uid) {
         final Matcher matcher = pattern.matcher(uhUuid);
         return matcher.matches() || uhUuid.equals(uid);
     }
+
+    public boolean isDepartmentAccount(String uhUuid, String uid) {
+        logger.info("isDepartmentAccount; returns if uhUuid match uid");
```

Comment:
For this logger message as well as in the isValidUhUuid() function, could you have the message show the uhUuid and uid passed in? This piece of code should work: 
`logger.info(String.format("isDepartmentAccount; uhUuid: %s; uid: %s;, uhUuid, uid));`
Thanks!

Reply:
The code provided works, thank you.